<launch>
  <node pkg="perception" type="static_tf_broadcaster_node" name="static_tf_broadcaster" output="screen" />

  <!-- Nodelet 管理器 -->
  <node pkg="nodelet" type="nodelet" name="depth_image_proc_manager" args="manager" output="screen" />

  <!-- Depth to PointCloud -->
  <node pkg="nodelet" type="nodelet" name="depth_to_cloud"
        args="load depth_image_proc/point_cloud_xyz depth_image_proc_manager" output="screen">
    <remap from="image_rect" to="/Unity_ROS_message_Rx/OurCar/Sensors/DepthCamera/image_raw"/>
    <remap from="camera_info" to="/Unity_ROS_message_Rx/OurCar/Sensors/DepthCamera/camera_info"/>
    <remap from="points" to="/depth/points"/>
    <param name="output_frame_id" value="OurCar/Sensors/DepthCamera"/>
  </node>
</launch>